samples/azure/azure-service-fabric-routing/Core_6/CandidateVoteCount/CandidateVotes.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.Threading.Tasks;
+using NServiceBus;
+
+public class CandidateVotes :
```

Comment:
My OCD kicks in, can it have the postfix Saga or otherwise have a name static something about that it does?